### PR TITLE
CI: pull rspec workflow out of Docker image

### PR DESCRIPTION
**Pull request by mockdeep**

### CODE REVIEW

**Comment by managedkaos** [comment]
@mockdeep  if you pull this one out of draft, I'm ready to move forward with it.

**Review by managedkaos — COMMENTED** [review]
@mockdeep Any reason why the codeclimate workflow was removed?  Just wondering...

**Comment by mockdeep** [comment]
@managedkaos codeclimate is already reporting from its own infrastructure, so it seemed redundant to also be running it on Github Actions.

code-climate

File: .env.test [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Sample .env file
```
Comment (rlgreen91) [inline .env.test:1]: As a heads up, we already have `.env.sample`, so if you need to push up any changes you can toss this file.

**Review by rlgreen91 — COMMENTED** [review]
LGTM, @managedkaos just let me know when you approve it and I'll merge.

**Review by managedkaos — APPROVED** [review]
LGTMe!

**Pull request merged**